### PR TITLE
Correctly log elapsed milliseconds in `CentralDogmaBeanFactory`

### PR DESCRIPTION
**Pull request by KarboniteKream**
Motivation:

When the elapsed time in `CentralDogmaBeanFactory#get()` is logged, it prints nanoseconds with `ms` unit.

Modifications:

- Convert the elapsed time to milliseconds before logging.

Result:

The logs correctly display the elapsed time of obtaining the initial value.

### CODE REVIEW

**Review by jrhee17 — APPROVED** [review]
Thanks! Separately, we probably need to fix our CI 🙇

**Review by minwoox — APPROVED** [review]
Thanks! 👍 👍 👍

File: client/java/src/main/java/com/linecorp/centraldogma/client/updater/CentralDogmaBeanFactory.java [hunk 1]
```diff
@@ -328,7 +328,7 @@ public <T> T get(T defaultValue, Class<T> beanType, Consumer<T> changeListener,
                     throw ex;
                 }
             }
-            final long elapsedMillis = System.nanoTime() - t0;
+            final long elapsedMillis = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - t0);
```
Comment (ikhoon) [inline client/java/src/main/java/com/linecorp/centraldogma/client/updater/CentralDogmaBeanFactory.java:331]: 😱

**Review by ikhoon — APPROVED** [review]
Nice catch! 👍👍

**Comment by minwoox** [comment]
@KarboniteKream 👍 👍 👍

**Pull request merged**